Makefile

```diff
@@ -9,6 +9,7 @@ PREFIX     ?= /usr/local
 INSTALLDIR ?= $(DESTDIR)$(PREFIX)
 MANDIR     ?= $(INSTALLDIR)/share/man
 DOCDIR     ?= $(INSTALLDIR)/share/uzbl/docs
+LIBDIR     ?= $(INSTALLDIR)/lib/uzbl
```

Comment:
This should include `$(LIB_SUFFIX)` so that Fedora and Debian can insert their suffixes (`64` for Fedora and `/x86_64-linux-foo` for Debian).